pkg/api/mlflow/controller/artifact.go

```diff
@@ -26,3 +33,47 @@ func (c Controller) ListArtifacts(ctx *fiber.Ctx) error {
 	log.Debugf("artifactList response: %#v", resp)
 	return ctx.JSON(resp)
 }
+
+// GetArtifact handles `GET /artifacts/get` endpoint.
+func (c Controller) GetArtifact(ctx *fiber.Ctx) error {
+	req := request.GetArtifactRequest{}
+	if err := ctx.QueryParser(&req); err != nil {
+		return api.NewBadRequestError(err.Error())
+	}
+	log.Debugf("GetArtifact request: %#v", req)
+
+	artifact, err := c.artifactService.GetArtifact(ctx.Context(), &req)
+	if err != nil {
+		return err
+	}
+
+	filename := filepath.Base(req.Path)
+	ctx.Set("Content-Type", common.GetContentType(filename))
+	ctx.Set("Content-Disposition", fmt.Sprintf("attachment; filename=%s", filename))
+	ctx.Set("X-Content-Type-Options", "nosniff")
+	ctx.Context().Response.SetBodyStreamWriter(func(w *bufio.Writer) {
+		defer artifact.Close()
+
+		start := time.Now()
+		if err := func() error {
+			bytesWritten, err := io.CopyBuffer(w, artifact, make([]byte, 4096))
+			if err != nil {
+				return fmt.Errorf("unable to copy artifact Reader to output stream: %w", err)
```

Comment:
can we try to use next format of error message -> `error doing something`. in that case it is much much easier when you search over the log, you always know that everything related to error star with `error` and it will be nice to still use `eris` since it add more information into `error` context.

Reply:
can do

Reply:
but of course, it is obvious an error message pertains to error